packages/camel-catalog/kaoto-camel-catalog-maven-plugin/src/test/java/io/kaoto/camelcatalog/CamelCatalogProcessorTest.java

```diff
@@ -18,7 +18,7 @@
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.fasterxml.jackson.databind.node.ArrayNode;
 import com.fasterxml.jackson.databind.node.ObjectNode;
-import org.apache.camel.dsl.yaml.CamelYamlRoutesBuilderLoader;
+import org.apache.camel.dsl.yaml.YamlRoutesBuilderLoader;
```

Comment:
The original file was removed here: https://github.com/apache/camel/pull/13135/files#diff-9e46a8c92d107cc1b4cd595bff2e454dd984d2fe3602ed0e39c756bff1ca8f8a